.ansible-lint

```diff
@@ -8,8 +8,9 @@ exclude_paths:
 # Mock modules or roles in order to pass ansible-playbook --syntax-check
 mock_modules:
   - zuul_return
-  - fake_namespace.fake_collection.fake_module
   # note the foo.bar is invalid as being neither a module or a collection
+  - fake_namespace.fake_collection.fake_module
+  - fake_namespace.fake_collection.fake_module.fake_submodule
```

Comment:
```suggestion
  - fake_namespace.fake_collection.fake_module
```
I would really avoid advertising that you can have further dots in module names ;-)

Reply:
Is not so much about advertising desired but listing it here assures that we do not choke when we encounter it (there it at least one test making use of it).

Reply:
In that case, at least use correct terms: `fake_namespace.fake_collection.fake_directory.fake_module`